homeassistant/components/velbus/__init__.py

```diff
@@ -118,7 +115,10 @@ def unique_id(self):
             serial = self._module.get_module_address()
         else:
             serial = self._module.serial
-        return f"{serial}-{self._channel}"
+        if self._is_counter:
```

Comment:
Same as above. Overwrite this property in the sensor entity class and added the counter string to the unique_id returned from the super unique_id property.

Reply:
Fixed in this [commit](https://github.com/home-assistant/home-assistant/pull/31165/commits/33104b2cc59eeb8bcd497d9672f0dda7de8f1159)